notifiers/providers/email.py

```diff
@@ -121,6 +127,15 @@ def _build_email(self, data: dict) -> MIMEMultipart:
         email.attach(MIMEText(data['message'].encode('utf-8'), content_type, _charset='utf-8'))
         return email
 
+    def _add_attachments(self, data: dict, email) -> MIMEMultipart:
+        for attachment in data['attachments']:
+            file = open(attachment, 'rb')
```

Comment:
Check out what I did with mailgun: https://github.com/liiight/notifiers/blob/develop/notifiers/providers/mailgun.py#L239

That wat you verify the files are valid first. Note the `_validate_data_dependencies` hook. Then you can access the Path object much error nicer like so:
```python
part = attachment.read_bytes()
```

Reply:
So I'm good on everything else I think.  Here should I replace line 132 like this:

```python
from pathlib import Path
...
def _add_attachments(self, data: dict, email) -> MIMEMultipart:
      for attachment in data['attachments']:
            file = Path(attachment).read_bytes()
            part = MIMEApplication(file)
            part.add_header('Content-Disposition', 'attachment', filename=attachment)
            email.attach(part)
      return email
```

Reply:
Yeah, looks good